src/content/cloud/okteto-cli.mdx

```diff
@@ -57,36 +58,72 @@ dev:
       - .:/usr/src/app
 ```
 
-This example shows how to `build` the container image of the **api** microservice, how to `deploy` the application using its helm chart,
-and how to `develop` on the **api** microservice by synchronizing the local filesystem on the container path `/usr/src/app`.
+This example shows how to [build](reference/manifest.mdx/#build-object-optional) the container image of the **api** microservice, how to [deploy](reference/manifest.mdx/#deploy-object-optional) the application using its helm chart,
+and how to [develop](reference/manifest.mdx/#dev-object-optional) on the **api** microservice by synchronizing the local filesystem on the container path `/usr/src/app`.
 If you want to learn more about what all you can configure in this manifest, head over to the [Okteto Manifest Reference](reference/manifest.mdx).
 
 ### Built-in Environment Variables
 
-The installation job also populates the following environment variables. You can refer to this in your `deploy` commands if needed:
+#### Default Environment Variables
+
+The following environment variables are automatically populated to be referred in your Okteto Manifest or deploy/destroy commands:
+- `OKTETO_DOMAIN`: the domain where Okteto exposes your application endpoints.
 - `OKTETO_NAMESPACE`: the namespace where your application is going to be installed.
+- `OKTETO_USERNAME`: your username in Okteto.
+- `OKTETO_REGISTRY_URL`: the url of the [Okteto Registry](cloud/registry.mdx).
 - `OKTETO_GIT_BRANCH`: the name of the Git branch currently being deployed.
 - `OKTETO_GIT_COMMIT`: the SHA1 hash of the last commit of the branch.
-- `OKTETO_REGISTRY_URL`: the url of the [Okteto Registry](cloud/registry.mdx).
-- `BUILDKIT_HOST`: the address of the [Okteto Cloud buildkit instance](cloud/build.mdx). You can use it to build your images with any CLI tool that supports Buildkit builds.
 
-In order to refer to images in the `build` section of the Okteto Manifest, you can use the following environment variables:
-- `${OKTETO_BUILD_<service>_IMAGE}`: the full image reference
-- `${OKTETO_BUILD_<service>_REGISTRY}`: the registry URL where the image was pushed
-- `${OKTETO_BUILD_<service>_REPOSITORY}`: the name of the image that was pushed
-- `${OKTETO_BUILD_<service>_SHA}`: The latest tag and the SHA of the image
+#### Build Environment Variables
```

Comment:
The name 'Build Environment Variables' doesn't clearly explain what they represent. Do we need a separate section?  
```suggestion
#### Built-in Environment Variables for Images stored in the Okteto Registry
```